python/Ganga/Core/MonitoringComponent/Local_GangaMC_Service.py

```diff
@@ -1158,11 +1158,15 @@ def makeCredChecker(self, credObj):
         def credChecker():
             log.debug("Checking %s." % getName(credObj))
             try:
-                s = credObj.renew()
-            except Exception as msg:
+                credObj.renew()
+            except CredentialRenewalError:
```

Comment:
The codebase has been moving to be better laid out imo. We already put all of the interfaces in `Ganga.GPIDev.Adapters` and exceptions in `Ganga.Core.exception`. This has been the way we've been heading for a while by moving other files around so that it's easy to know where things are.